Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CPUPercenageGauge.cs

```diff
@@ -37,6 +37,15 @@ public CPUPercenageGauge(string name, ICounterValue value)
         /// <returns>The value of the target metric.</returns>
         public double GetValueAndReset()
         {
+            return this.Collect();
+        }
+
+        /// <summary>
+        /// Returns the percentage of the CPU process utilization time with respect to the total duration.
+        /// </summary>
+        /// <returns>The value of the target metric.</returns>
+        protected virtual double Collect()
```

Comment:
There is a typo in this file/class name. CPUPercenageGauge to CPUPercentageGauge - can you fix that as well if time permits.